src/study_groups.py

```
@@ -0,0 +1,712 @@
+import re
+from typing import Any, Awaitable, Callable, Dict, Iterable, List, Optional, Tuple
```

Flake8 (7.3.0)

[error] 'typing.Awaitable' imported but unused

(F401)